test/deps/Project.toml

```diff
@@ -14,6 +13,10 @@ Skix = "3db6da90-6ed3-11ee-0779-f549c8e3e90d"
 [extensions]
 Ext = ["Bex_jll", "Skix"]
 
+[extras]
+Baz = "ea10d353-3f73-51f8-a26c-33c1cb351aa5"
```

Comment:
Any particular reason for moving Baz from `[deps]` to `[extras]`?

Reply:
I'm just coming back to this now. I realize now that if we remove `Baz` from `[deps]`, then `[deps]` doesn't contain any direct non-stdlib non-JLL dependencies (because `Foobar_jll` is a JLL, and `LinearAlgebra` is a stdlib), which obviously is a codepath that we want to make sure to cover.

Can you move Baz back to `[deps]`, and then have an additional dep in `[extras]`?